ServiceManager/GUI/main_window.py

```diff
@@ -0,0 +1,629 @@
+        self.service_log_txt.horizontalScrollBar().setValue(old_h_scrollbar_value)
+
+    def clear_service_log(self):
+        self.service_log_txt.clear()
+
+    def log_scroll_to_bottom(self):
+        self.service_log_txt.verticalScrollBar(). \
+            setValue(self.service_log_txt.verticalScrollBar().maximum())
+
+    def open_service_log(self):
+        debug_file_path = Settings.Service.Logging.get_debug_log_path()
+        if os.path.exists(debug_file_path):
+            os.startfile(debug_file_path)
+        else:
+            QMessageBox.critical(self, 'Service Log File Not Found',
+                                       'The service log file was not found.\n'
+                                       f'"{debug_file_path}" does not exist.',
+                                       QMessageBox.Ok)
+            return
+
+    def update_log_displayed_lines_no(self, value):
+        self.thread_service_log.set_displayed_lines_no(value)
+
+    def update_log_font_size(self, value):
+        self.service_log_txt.setStyleSheet(f'font-size: {value}pt;')
+
+    def update_service_log_btns(self, enabled: bool):
+        self.service_log_file_open_btn.setEnabled(enabled)
+        self.service_log_scroll_down_btn.setEnabled(enabled)
+        self.service_log_show_lines_spinbox.setEnabled(enabled)
+    # endregion
+
+    # region Service Status
+    def update_status_title(self, text: str):
+        self.service_status_title.setText(text)
+
+    def update_status_style(self, stylesheet: str):
+        self.service_status_title.setStyleSheet(stylesheet)
+
+    def update_service_details(self, service: dict):
+        self.service_details_name.setText(service['name'])
+        self.service_details_status.setText(service['status'])
+        self.service_details_pid.setText(f"{service['pid']}")
+        self.service_details_startup.setText(service['start_type'])
+        self.service_details_description.setText(service['description'])
+        self.service_details_username.setText(service['username'])
+        self.service_details_binpath.setText(service['binpath'])
+
+    def update_service_control_btns(self, service_status):
+        if service_status == 'running':
+            self.btn_service_start.setEnabled(False)
+            self.btn_service_stop.setEnabled(True)
+            self.btn_service_restart.setEnabled(True)
+        elif service_status == 'stopped':
+            self.btn_service_start.setEnabled(True)
+            self.btn_service_stop.setEnabled(False)
+            self.btn_service_restart.setEnabled(False)
+        elif service_status is None or service_status == 'not found':
+            self.btn_service_start.setEnabled(False)
+            self.btn_service_stop.setEnabled(False)
+            self.btn_service_restart.setEnabled(False)
+        else:
+            self.btn_service_start.setEnabled(True)
+            self.btn_service_stop.setEnabled(True)
+            self.btn_service_restart.setEnabled(True)
```

Comment:
**should**:
```suggestion
        stopped = service_status == psutil.STATUS_STOPPED
        running = service_status == psutil.STATUS_RUNNING
        self.btn_service_start.setEnabled(stopped)
        self.btn_service_stop.setEnabled(running)
        self.btn_service_restart.setEnabled(running)
```

I think this covers all eventualities? If it's neither stopped or running e.g. an error it will set them all to False and I'm not sure why you want to have the all True situation?